src/Blameable/BlameableListener.php

```diff
@@ -11,6 +11,8 @@
  *
  * @author Gediminas Morkevicius <gediminas.morkevicius@gmail.com>
  * @license MIT License (http://www.opensource.org/licenses/mit-license.php)
+ *
+ * @final since gedmo/doctrine-extensions 3.x
```

Comment:
should mention the actual minor version rather than saying `3.x`

Reply:
I'm performing these replacements during the release process, as we don't know yet if the next minor release will include this change. Take a look at #2296 for an example.